client/homescreen/activity-panel/orders/index.js

```diff
@@ -227,7 +227,7 @@ function OrdersPanel( { countUnreadOrders, orderStatuses } ) {
 		} ),
 		[ orderStatuses ]
 	);
-	const { orders, isRequesting, isError } = useSelect( ( select ) => {
+	const { orders = [], isRequesting, isError } = useSelect( ( select ) => {
```

Comment:
I think this should actually be fixed in the early return (line 253) by adding `orders: [],`.

I had to scan this a few times to understand how orders could be undefined before spotting the early return. Making the returns consistent would be easier to comprehend.

Reply:
@becdetat Good suggestion 👍 Updated in https://github.com/woocommerce/woocommerce-admin/pull/7506/commits/a7df925cb4bf81a393f7e19e40ede55a2c4f5f40

Reply:
Added `orders: []` in the second early return as well https://github.com/woocommerce/woocommerce-admin/pull/7506/commits/ad24200f1deb49f248859603a511538e2b7da234